### DISCUSSION TITLE
Fix CSS file name

### DISCUSSION THREAD

**Discussion by levipadre** [vercel/next.js#71154] [Help]
### Summary

I want to generate an additional CSS file, like rte.css for CMS, so I can use it on the frontend and in the CMS with a Rich Text Editor.
At the moment is impossible, because Nextjs generates the CSS files with a random name.

Is it possible to create this file somehow so I can have two scss files to import to layout.tsx, e.g. style.scss and rte.scss?

### Additional information

```js
Operating System:
      Platform: darwin
      Arch: x64
      Version: Darwin Kernel Version 23.6.0: Mon Jul 29 21:13:00 PDT 2024; root:xnu-10063.141.2~1/RELEASE_X86_64
    Binaries:
      Node: 18.20.4
      npm: 10.8.3
      Yarn: 1.22.19
      pnpm: 9.7.0
    Relevant Packages:
      next: 13.5.1
      eslint-config-next: N/A
      react: 18.2.0
      react-dom: 18.2.0
      typescript: 5.2.2
    Next.js Config:
      output: N/A
```

**Comment by icyJoseph** [comment]
I think you can do something similar to what they do here, https://github.com/vercel/next.js/discussions/65856#discussioncomment-9749287
